src/viewProviders/RokuAutomationViewViewProvider.ts

```diff
@@ -87,6 +93,7 @@ export class RokuAutomationViewViewProvider extends BaseRdbViewProvider {
     }
 
     private isRecording = false;
+    private selectedConfig;
```

Comment:
Let's add a specific typescript type for this (it's currently defaulting to `any`)

Reply:
done